internal/tools/aws/vpc.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package aws
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+// GetCIDRByVPCTag creates a record in Route53 for a public domain name.
+func (a *Client) GetCIDRByVPCTag(vpcTagName string, logger log.FieldLogger) (string, error) {
+	vpcInput := ec2.DescribeVpcsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String(VpcNameTagKey),
+				Values: []*string{aws.String(vpcTagName)},
+			},
+		},
+	}
+
+	vpcOut, err := a.Service().ec2.DescribeVpcs(&vpcInput)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to fetch the VPC information using tag %s", vpcTagName)
+	}
+
+	if len(vpcOut.Vpcs) > 1 {
+		return "", errors.New(fmt.Sprintf("expected just one VPC in the list, got %d", len(vpcOut.Vpcs)))
```

Comment:
If we don't ensure that we have at least one VPC here it will panic on line 37.

```suggestion
	if len(vpcOut.Vpcs) != 1 {
		return "", errors.New(fmt.Sprintf("expected exactly one VPC in the list, got %d", len(vpcOut.Vpcs)))
```